vm_server/accept/server.py

```diff
@@ -18,28 +19,68 @@
 
 sem = threading.Semaphore()
 
-def remove_execute_dir():
-  """Deletes the execute directory if it exists"""
+def remove_execute_dir(task_response):
+  """Deletes the execute directory if it exists
+
+  Args:
+    task_response: an object of TaskResponse() that will be sent back
+  """
+  logging.debug("Removing execute directory")
   dirpath = Path("..\\execute")
-  if dirpath.exists() and dirpath.is_dir(): # delete leftover files
-    shutil.rmtree(dirpath)
+  try:
+    if dirpath.exists() and dirpath.is_dir(): # delete leftover files
+      shutil.rmtree(dirpath)
+  except Exception as exception:  #catch errors if any
```

Comment:
This is probably the minutest of nit picks that I have ever done. (critique automatically points this out) - extra space before comment.

Reply:
Added spaces after hash :)